### PR TITLE
feat: backup settings

### PR DESCRIPTION
**Pull request by nuanyang233**
## Description

Closes #MF-4451

Removed useless code from the dashboard, like `Personas` and `Settings`.

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

File: packages/mask/src/utils/fetcher.ts [hunk 1]
```diff
@@ -42,6 +42,7 @@ function canAccessAsContent(url: string) {
     )
         return true
 
+    if (target.origin.includes('maskbook-backup')) return true
```
Comment (Jack-Works) [inline packages/mask/src/utils/fetcher.ts:47]: can you give an example and write it as a comment in the code?

File: packages/mask/src/extension/popups/pages/Settings/index.tsx [hunk 2]
```diff
@@ -136,8 +134,39 @@ const Settings = memo(function Settings() {
...
+    }, [user])
+
     const { data } = useSupportedSites()
 
+    const handleOpenDashboard = useCallback(async (route: DashboardRoutes) => {
+        await browser.tabs.create({
+            active: true,
+            url: browser.runtime.getURL(`/dashboard.html#${route}`),
+        })
+        if (navigator.userAgent.includes('Firefox')) {
```
Comment (guanbinrui) [inline packages/mask/src/extension/popups/pages/Settings/index.tsx]: `Sniffings.is_firefox`

File: packages/mask/dashboard/components/CountryCodePicker/index.tsx [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+                            onClick={() => {
+                                onClose(data.dial_code)
+                            }}
+                            key={data.code}
+                            className={classes.listItem}
+                            autoFocus={selected}
+                            selected={selected}>
+                            <ListItemIcon className={classes.listItemIcon}>
+                                <img src={icon} className={classes.icon} />
+                                {/* {createElement(icon, { className: classes.icon })} */}
```
Comment (guanbinrui) [inline packages/mask/dashboard/components/CountryCodePicker/index.tsx]:
```suggestion
```

File: packages/mask/src/utils/fetcher.ts (hunk 1 above)
Comment (Jack-Works) [reply packages/mask/src/utils/fetcher.ts:47]: this looks... pretty unsafe. can we have a stable domain??

File: packages/mask/background/tasks/NotCancellable/OnInstall.ts [hunk 4]
```diff
@@ -16,5 +16,19 @@ browser.runtime.onInstalled.addListener(async (detail) => {
     } else if (detail.reason === 'update') {
         const groups = await connect.getOriginsWithoutPermission()
         if (groups.length) openWelcome()
+        if ((globalThis as any).localStorage) {
+            const localStorage = (globalThis as any).localStorage
+            const backupPassword = localStorage.getItem('backupPassword')
```
Comment (Jack-Works) [inline packages/mask/background/tasks/NotCancellable/OnInstall.ts:21]: you don't delete old data after migrate?

**Pull request merged**